brainstormforce/astra
# AST-1199 - When "Display site title" option is disabled still we see the heading tags

### Description
In site title & logo options if we have the display site title option disabled then still we see the H1 tags.

### Screenshots
https://drive.google.com/file/d/1jkE5SqvihwynQ7bNgj4I3eU3f0xZV9Rf/view?usp=drivesdk

### Types of changes

Bug fix (non-breaking change which fixes an issue) 



### How has this been tested?
- Go to site title & logo settings
- Disable the “Display site title” option
- Check for the error in the console (ref: see the above screenshots)
- The tags should be hidden if the option is disabled.

### Checklist:
